search/elastic.py

```diff
@@ -616,26 +616,41 @@ def search(self,
                 }
             )
         """
-
-        log.debug("searching index with %s", query_string)
+        query_strings = [] if not query_strings else query_strings
+        query_strings = [query_strings] if isinstance(query_strings, (str, unicode)) else query_strings
+        log.debug('searching index with terms [{}]', ','.join(query_strings))
```

Comment:
@BarryPaneer We can remove the log if it's not necessary.

Reply:
done